src/uk/ac/open/audio/streaming/StreamPlayerUI.java

```diff
@@ -188,6 +196,47 @@ public void actionPerformed(ActionEvent e)
 				}
 			}
 		});
+
```

Comment:
I think there might be some issues with whitespace indent as well as with the brace positioning in the below code.